tests/vc-generator/index.js

```diff
@@ -0,0 +1,112 @@
+/*!
+ * Copyright 2023 Digital Bazaar, Inc.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+import * as vc from '@digitalbazaar/vc';
+import {DataIntegrityProof} from '@digitalbazaar/data-integrity';
+import {documentLoader} from './documentLoader.js';
+import {getMultikeys} from './key-gen.js';
+import {getSuite} from './cryptosuites.js';
+import {klona} from 'klona';
+
+/**
+ * Issues test data locally and then returns a Map
+ * with the test data.
+ *
+ * @param {object} options - Options to use.
+ * @param {object} options.credential - An unsigned VC.
+ * @param {string} options.suite - A cryptosuite id.
+ * @param {Array<string>} options.mandatoryPointers - An optional list of
+ *   json pointers.
+ * @param {Array<string>} options.keyTypes - A Set of keyTypes to issue with.
+ *
+ * @returns {Promise<Map<string, object>>} Returns a Map <keyType, vc>.
+ */
+export async function issueTestData({
+  credential,
+  suite,
+  mandatoryPointers,
+  keyTypes = ['P-256']
+}) {
+  const results = new Map();
+  const keys = await getMultikeys({keyTypes});
+  for(const [keyType, {signer, issuer}] of keys) {
+    const _vc = await issueCredential({
+      credential,
+      issuer,
+      signer,
+      suite,
+      mandatoryPointers
+    });
+    results.set(keyType, _vc);
+  }
+  return results;
+}
+
+export async function issueCredential({
+  credential,
+  issuer,
+  signer,
+  suite,
+  mandatoryPointers = []
+}) {
```

Comment:
Question: why do some functions provide jsdoc and not others?

Reply:
Time? And docs aren't currently generated (and therefore not consulted), so there's no pressure to keep it up.

If you'd find it educational/useful to add some as you poke around, feel free! It will likely point you to some obvious refactoring needs.

Reply:
technically issueCredential is a helper function for `issueTestData` which already contains jsdoc for it, I guess I just felt like it seems redundant to document them here when we have pressing other matters.